cloudify_ansible/tests/test_workflow.py

```diff
@@ -26,68 +36,245 @@
+        reason='This test requires you to run "vagrant up". '
+               'And export TEST_ZPLAYS=true')
+    def test6_clearwater_blueprint(self):
+        load_new_vagrant_env(['ellis',
+                              'bono',
+                              'sprout',
+                              'homer',
+                              'homestead',
+                              'ralf',
+                              'bind'])
+        with open(PRIVATE_KEY_DIR.format('ellis'), 'r') as infile:
+            ellis_private_key = infile.read()
+        with open(PRIVATE_KEY_DIR.format('bono'), 'r') as infile:
+            bono_private_key = infile.read()
+        with open(PRIVATE_KEY_DIR.format('sprout'), 'r') as infile:
+            sprout_private_key = infile.read()
+        with open(PRIVATE_KEY_DIR.format('homer'), 'r') as infile:
+            homer_private_key = infile.read()
+        with open(PRIVATE_KEY_DIR.format('homestead'), 'r') as infile:
+            homestead_private_key = infile.read()
+        with open(PRIVATE_KEY_DIR.format('ralf'), 'r') as infile:
+            ralf_private_key = infile.read()
+        with open(PRIVATE_KEY_DIR.format('bind'), 'r') as infile:
+            bind_private_key = infile.read()
+        cfy_local = local.init_env(
+            _clearwater_blueprint,
+            'test6_clearwater_blueprint',
+            inputs={
+                'ellis_private_key': ellis_private_key,
+                'bono_private_key': bono_private_key,
+                'sprout_private_key': sprout_private_key,
+                'homer_private_key': homer_private_key,
+                'homestead_private_key': homestead_private_key,
+                'ralf_private_key': ralf_private_key,
+                'bind_private_key': bind_private_key,
+            },
+            ignored_modules=IGNORED_LOCAL_WORKFLOW_MODULES)
+        cfy_local.execute('install', task_retries=0)
+        self.assertIn(
+            'result',
+            cfy_local.storage.get_node_instances(
+                'ellis')[0].runtime_properties.keys())
+        self.assertIn(
+            'result',
+            cfy_local.storage.get_node_instances(
+                'bono')[0].runtime_properties.keys())
+        self.assertIn(
+            'result',
+            cfy_local.storage.get_node_instances(
+                'sprout')[0].runtime_properties.keys())
+        self.assertIn(
+            'result',
+            cfy_local.storage.get_node_instances(
+                'homer')[0].runtime_properties.keys())
+        self.assertIn(
+            'result',
+            cfy_local.storage.get_node_instances(
+                'homestead')[0].runtime_properties.keys())
+        self.assertIn(
+            'result',
+            cfy_local.storage.get_node_instances(
+                'ralf')[0].runtime_properties.keys())
+        self.assertIn(
+            'result',
+            cfy_local.storage.get_node_instances(
+                'bind')[0].runtime_properties.keys())
```

Comment:
@EarthmanT I think we can re-write all the above assertion as the following 

```
instances = ['ellis', 'bono', 'sprout', 'homer', 'homestead', 'ralf', 'bind']
for item in instances:
    self.assertIn('result', cfy_local.storage.get_node_instances(item)[0].runtime_properties.keys())

```

Reply:
Sure ok.